test/fixture/pages/cookie.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <pre style="display: none">_req:{{ reqCookie }}</pre>
+    <p>Pass: {{ pass }}</p>
+  </div>
+</template>
+
+<script>
+export default {
+  async asyncData ({ app }) {
+    const reqCookie = (await app.$axios.$get('/cookie')) + ''
```

Comment:
```suggestion
    const reqCookie = (await app.$axios.$get('/cookie', {
      headers: { Accept: 'application/json' }
    })) + ''
```

Reply:
If we add a local config it will not be merged with the default config and will not work with the current version, but if you set axios to `0.19.2` it works. I think it was changed in this PR https://github.com/axios/axios/pull/2844